### ISSUE TITLE
[Tool] Add more options to select the location of a new project

### ISSUE THREAD

**Issue by FredericHirschmueller** [open-plan-tool/gui#18]
When creating a new project, it is necessary to choose a location. Currently it is only possible to select a location through the displayed map. It would be nice to add more options to select a location, like:

- zip code
- coordinates
- address
- current location of the used device

**Comment by Bachibouzouk** [comment]
I would say it is a nice-to-have as the user need to do it only once per project

**Comment by FredericHirschmueller** [comment]
I agree that we should not focus too much on implementing many options to choose the location. However, it would be nice to have at least one more option since some stakeholder had problems choosing the location on a map. For example the map is not displayed when using open_plan on the phone, hence it is not possible to create a project on the phone.
